crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -568,25 +585,17 @@ module sui::validator_set {
     fun calculate_total_stake_and_quorum_threshold(validators: &vector<Validator>): (u64, u64, u64) {
         let validator_state = 0;
         let delegate_stake = 0;
+        let total_voting_power = 0;
```

Comment:
This function no longer makes sense in that it's mixing the sum of stake and voting power threshold together, that's strange.
We should separate computations based on stake and computations based on voting power